models/sepc/builder.py

```diff
@@ -0,0 +1,126 @@
+import mxnet as mx
+import mxnext as X
+from models.NASFPN.builder import RetinaNetNeck, RetinaNetNeckWithBN
+from models.sepc.sepc_neck import SEPCFPN
+from utils.patch_config import patch_config_as_nothrow
+from models.NASFPN.builder import RetinaNetHeadWithBN
+import math
+
+
+class RetinaNetNeckWithBNWithSEPC(RetinaNetNeckWithBN):
+    def __init__(self, pNeck, pSEPC):
+        super().__init__(pNeck)
+        self.psepc = patch_config_as_nothrow(pSEPC)
+        self.neck_with_sepc = None
+        stride, pad_sizes = pSEPC.stride, pSEPC.pad_sizes
+        self.feat_sizes = [[math.ceil(pad_sizes[0]/stride[i]), math.ceil(pad_sizes[1]/stride[i])] for i in range(len(stride))]
```

Comment:
The calculation of feature size does not generalize to different input size

Reply:
Hi,  here I handle the feature size in such a simple way because it's necessary for my implementation of ibn_func. How can I generalize the calculation of feature size or is another implementaion of ibn_func possible? Thanks.

Reply:
It is ok that you want to handle the feature size in this simple way, but maybe some assertion and error message to reminder the user when they give input sizes which are not dividable by the total stride.

Reply:
Hi, I have add assertion error message for case when ibn is used but input sizes are not all dividable by the strides.

Reply:
Hi, I have add warning message for case when ibn is used in SEPC but the input sizes are not dividable by the strides.